### ISSUE TITLE
Integrate with ddb-importer to optionally add icons on import

### ISSUE THREAD

**Issue by Oshcakes** [jbhaywood/5e-statblock-importer#19]
Hey there! I reached out to Mr Primate and he made the API available from his module.

See this:  https://github.com/MrPrimate/ddb-importer/blob/main/FAQ.md#can-i-use-the-icon-mapping-api

Thanks

**Comment by jbhaywood** [comment]
Interesting. I'll take a look when I get a chance. I should be able to get icons for the standard weapons and such directly from the compendiums that come as part of the 5e system. I'd rather avoid tying this to another module since you never know when it might change or go away completely.

**Comment by Oshcakes** [comment]
Yeah this uses the core icons. The API just maps items, features, spells, etc to the existing icons based on the keyword and/or type. So all of that mind numbing work of matching icons is already done. (I had a hand in some of it myself)

Example:
```
await DDBImporter.getIconPath(item)
'systems/dnd5e/icons/skills/weapon_18.jpg'
```

I very much doubt this goes away anytime soon as this is backed by a very healthy patreon and the code base is updated often. Plus Mr P. is a cool dude :)
